CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 2.3.0
```

Comment:
I want to merge one more RP before release. It's about bug with selected text in FF38-39.